contrib/opentelemetry/tracing_interceptor.go

```diff
@@ -41,6 +41,12 @@ type TracerOptions struct {
 	// DisableBaggage can be set to disable baggage propagation.
 	DisableBaggage bool
 
+	// EnableSpanLinks can be set to enable span links for spans disconnected from their parent span.
+	EnableSpanLinks bool
```

Comment:
Does this need to be an additional option? It seems like span links (an otel-specific feature?) could be added automatically if the disconnect continue as new workflows option is set.